packages/caliper-samples/network/fabric-v1.4.1/config-swarm/configtx.yaml

```diff
@@ -0,0 +1,135 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+Organizations:
+    - &OrdererOrg
```

Comment:
Does the '&' mean anything?

Reply:
Yeah, it's like a reference to reduce the redundant description. &xx are referred with *xx in other parts.